src/util.js

```diff
@@ -216,11 +216,10 @@ function murmurhash2_32_gc(str) {
 export const hashObject = (object /* : ObjectMap */) /* : string */ => murmurhash2_32_gc(JSON.stringify(object));
 
 
-const IMPORTANT_RE = /^([^:]+:.*?)( !important)?;$/;
-
-// Given a single style rule string like "a: b;", adds !important to generate
-// "a: b !important;".
-export const importantify = (string /* : string */) /* : string */ =>
-    string.replace(
-        IMPORTANT_RE,
-        (_, base) => base + " !important;");
+// Given a single style value string like the "b" from "a: b;", adds !important
+// to generate "b !important".
+export const importantify = (string /* : string */) /* : string */ => (
+    string[string.length - 10] === '!' && string.slice(-11) === ' !important'
```

Comment:
hehe this is great! Could you maybe add a comment explaining why this check looks so funky? Also maybe wrap these checks in parens because I can never remember the precedence of `?:`? :P